src/math/Interpolate.cc

```diff
@@ -157,7 +157,7 @@ ::gsl_interp_type const *styleToGslInterpType(Interpolate::Style const style) {
         case Interpolate::UNKNOWN:
             throw LSST_EXCEPT(pex::exceptions::InvalidParameterError,
                               "I am unable to make an interpolator of type UNKNOWN");
-        case Interpolate::NUM_STYLES:
+        default:
```

Comment:
What about:

         case Interpolate::NUM_STYLES:
              throw LSST_EXCEPT(pex::exceptions::LogicError,
                                str(boost::format("You can't get here: style == %") % style));
         default:
              abort();